core/src/main/scala/org/http4s/EntityEncoder.scala

```diff
@@ -192,6 +193,9 @@ trait EntityEncoderInstances extends EntityEncoderInstances0 {
   def chunkedEncoder[A](f: A => Channel[Task, Int, ByteVector], chunkSize: Int = 4096): EntityEncoder[A] =
     sourceEncoder[ByteVector].contramap { a => Process.constant(chunkSize).toSource.through(f(a)) }
 
+  implicit val multipartEncoder: EntityEncoder[Multipart] =
```

Comment:
There is a sharp edge here: entity encoder headers don't know the entity value, and thus can't render the boundary extension to the Content-Type, so this encoder doesn't render any headers at all.  Either we need to refactor that aspect of EntityEncoder, or consider whether this is an EntityEncoder at all.  (The decoder is cool.)